doc/modules/ROOT/pages/usage/misc.adoc

```diff
@@ -44,6 +44,26 @@ setting `nrepl.middleware.completion/*complete-fn*`.
 user=> (set! nrepl.middleware.completion/*complete-fn* my.namespace/my-completion)
 ----
 
+=== Options
+
+The default completion function accepts the `extra-meta` option. It allows clients to request additional metadata about completion items. For example, to request docstrings and argument lists for applicable completion items:
+
+[source,clojure]
+----
+;; -> client sends completion op
+{:op "completions"
+ :prefix "ma"
+ :ns "clojure.core"
+ :extra-meta ["arglists" "doc"]}
```

Comment:
The example seems wrong, as `:extra-meta` should in a map. As this is public API we might also use a more descriptive name like `extra-metadata` or `additional-metadata`. `Meta` is a bit too `meta`. 😆